Objects/typeobject.c

```diff
@@ -1004,6 +1004,14 @@ type_repr(PyTypeObject *type)
     return rtn;
 }
 
+PyObject *
+_PyType_DisabledNew(PyTypeObject *type, PyObject *args, PyObject *kwds)
+{
+    PyErr_Format(PyExc_TypeError, "cannot create '%.100s' instances",
```

Comment:
Please don't truncate type names, just use %s. PyErr_Format() allocates a buffer large enough.

Reply:
Sure. This was the old behaviour; I did not change it because I though it would be out of scope.

Reply:
Fixed in 61d26e8df3bddae789cfda82fc77fb64fa438709